app/views/claims/existing_session.html.erb

```diff
@@ -0,0 +1,21 @@
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-full">
+    <div class="govuk-panel govuk-panel--interruption">
+      <h1 class="govuk-heading-xl">
+        You have an application in progress for <%= policy_service_name(current_policy_routing_name) %>.
+      </h1>
+
+      <h2 class="govuk-heading-m govuk-!-margin-bottom-9">
+        If you start a new application then your progress will be lost.
+      </h2>
+
+      <p class="govuk-body">
+        <%= link_to "Continue application", claim_path(current_policy_routing_name, slug: session[:current_slug]), class: "govuk-button" %>
```

Comment:
This will send the user to the last question that they answered, not the last one they saw.

Reply:
Also, do we need to make the call to action clearer that "Continue application" means continue the one that is already in progress, and not continue with the application they have just clicked on to start?

Reply:
"Continue current application" makes more sense, agreed. Do we need to ask a content designer for the exact wording ("Continue with in-progress application"?)

This doesn't look to be taking the user back to the last answered question, just running through it now it definitely takes me to the last seen question, and (I think) `spec/features/switching_policies_spec.rb:35` checks this behaviour (the test has selected the QTS year but not current school, and expects to see the current school question).

Reply:
Yep, the `before_action` here stores the current slug: https://github.com/DFE-Digital/dfe-teachers-payment-service/blob/dffb5e01ddc95153c57e39a8fcb66bbce9932d88/app/controllers/claims_controller.rb#L6. I've rejigged the commits a bit now, so it should make things a bit clearer

Reply:
RE the call to action button text and elsewhere, yes, Faye should definitely be walked through this flow and asked to provide feedback on the content.

Reply:
> "Continue current application" makes more sense, agreed. Do we need to ask a content designer for the exact wording ("Continue with in-progress application"?)
> 
> This doesn't look to be taking the user back to the last answered question, just running through it now it definitely takes me to the last seen question, and (I think) `spec/features/switching_policies_spec.rb:35` checks this behaviour (the test has selected the QTS year but not current school, and expects to see the current school question).

Hmmm, I must have done something weird when testing this locally on my machine.